ui/src/app/shared/pipes/short-date-time.pipe.ts

```diff
@@ -6,20 +6,8 @@ import {Pipe, PipeTransform} from '@angular/core';
 })
 export class ShortDateTimePipe extends DatePipe implements PipeTransform {
 
-  isToday(date: Date): boolean {
-    if (date) {
-      let today: Date = new Date();
-      return date.getUTCDate() == today.getUTCDate() &&
```

Comment:
I read the issue. It sounds to me @kbergin was reporting a bug, which is that we were showing timestamps from the current day with and without the date. THe source of the bug is likely that we're using UTC time here rather than local time. I don't think @kbergin was requesting you remove the differential timestamp entirely

Reply:
I agree that @kbergin was not asking for the display to be simplified, but the complexity of logic that is required to figure out time zones just to display the day/time differently if it's today doesn't seem commensurate with actual value to the user.